lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
Agree in everything but in the `background` option. I would keep that in the same level as `timeZoneOffsetInMinutes` or `minuteInterval`.